base/sparse/cholmod.jl

```diff
@@ -1325,4 +1325,7 @@ function ishermitian(A::Sparse{Complex{Float64}})
     end
 end
 
+(*){Ti}(A::Symmetric{Float64,SparseMatrixCSC{Float64,Ti}}, B::SparseMatrixCSC{Float64,Ti}) = sparse(Sparse(A)*Sparse(B))
```

Comment:
The `sparse` and `Sparse` are confusing. I wonder if we can do better names. In any case this should not prevent this from getting merged - but thought I'd bring it up right here.